### PR TITLE
Removed MassCreateDetailSmall layout on Asset

### PR DESCRIPTION
**Pull request by D-Raoul**

### CODE REVIEW

File: client/modules/dam/src/views/asset/list.js [hunk 1]
```diff
@@ -57,9 +57,9 @@ Espo.define('dam:views/asset/list', 'views/list-tree',
             this.notify('Loading...');
             this.createView('massCreate', 'dam:views/asset/modals/edit', {
                 scope: 'Asset',
-                attributes: this.getCreateAttributes() || {},
+                attributes: {massCreate: true},
```
Comment (rratsun) [inline client/modules/dam/src/views/asset/list.js:60]: why did you remove this.getCreateAttributes()? I think it can be a problem.

File: client/modules/dam/src/views/asset/modals/edit.js [hunk 2]
```diff
@@ -69,21 +69,21 @@ Espo.define('dam:views/asset/modals/edit', 'views/modals/edit',
...
+            this.actionQuickCreate(true);
         },
 
-        actionQuickCreate(layout) {
+        actionQuickCreate(massCreate) {
             let options = _.extend({
                 model: this.model,
                 scope: this.scope,
-                attributes: {}
+                attributes: {massCreate}
```
Comment (rratsun) [inline client/modules/dam/src/views/asset/modals/edit.js:84]: Masscreate is a bool variable. So how it should work here?

Comment (D-Raoul) [reply client/modules/dam/src/views/asset/modals/edit.js:84]: it is a bool variable, if it is true, the files field will be shown in the layout and the file and name will not be shown
